From 6328e577badad093ad52dfb9d343628b661d3a14 Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
Date: Mon, 16 Jul 2018 13:17:37 -0500
Subject: [PATCH 1337/1626] ASoC: SOF: remove unused SOF_PLATFORM from
 Makefile/Kconfig

This option points to non-existent files, remove.

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
---
 sound/soc/sof/Kconfig  | 3 ---
 sound/soc/sof/Makefile | 2 --
 2 files changed, 5 deletions(-)

diff --git a/sound/soc/sof/Kconfig b/sound/soc/sof/Kconfig
index cec2ea9..ba54d82 100644
--- a/sound/soc/sof/Kconfig
+++ b/sound/soc/sof/Kconfig
@@ -4,9 +4,6 @@ config SND_SOC_SOF_PCI
 config SND_SOC_SOF_ACPI
 	tristate
 
-config SND_SOC_SOF_PLATFORM
-	tristate
-
 config SND_SOC_SOF
 	tristate "Sound Open Firmware Support"
 	default m
diff --git a/sound/soc/sof/Makefile b/sound/soc/sof/Makefile
index b3ba39d..b8f6c67 100644
--- a/sound/soc/sof/Makefile
+++ b/sound/soc/sof/Makefile
@@ -8,7 +8,6 @@ snd-sof-spi-objs := hw-spi.o
 
 snd-sof-pci-objs := sof-pci-dev.o
 snd-sof-acpi-objs := sof-acpi-dev.o
-snd-sof-platform-objs := sof-platform-dev.o
 snd-sof-nocodec-objs := nocodec.o
 
 obj-$(CONFIG_SND_SOC_SOF) += snd-sof.o
@@ -17,7 +16,6 @@ obj-$(CONFIG_SND_SOC_SOF_NOCODEC) += snd-sof-nocodec.o
 obj-$(CONFIG_SND_SOC_SOF_ACPI) += sof-acpi-dev.o
 obj-$(CONFIG_SND_SOC_SOF_PCI) += sof-pci-dev.o
 obj-$(CONFIG_SND_SOC_SOF_SPI) += sof-spi-dev.o
-obj-$(SND_SOC_SOF_PLATFORM) += sof-platform-dev.o
 
 obj-$(CONFIG_SND_SOC_SOF_SPIDSP) += snd-sof-spi.o
 
-- 
2.7.4

